requirements/base.txt

```diff
@@ -232,16 +231,14 @@ six==1.16.0
     #   bleach
     #   click-repl
     #   flask-talisman
-    #   holidays
     #   isodate
     #   jsonschema
     #   polyline
     #   prison
     #   pyrsistent
     #   python-dateutil
-    #   sqlalchemy-utils
     #   wtforms-json
-slack_sdk==3.18.3
```

Comment:
Did something change here?

Reply:
The `-` for `_`. ;)

Reply:
ah good eye

Reply:
Good catch. Rant: I always cringe at the fact that `-` and `_` can be used interchangeably for Python deps.. 😡